Kickstarter-iOS/Views/Cells/SettingsTableViewCell.swift

```diff
@@ -28,15 +27,6 @@ final class SettingsTableViewCell: UITableViewCell, ValueCell, NibLoading {
       |> settingsArrowViewStyle
       |> UIImageView.lens.isHidden
       .~ !cellType.showArrowImageView
-
-    _ = detailLabel
```

Comment:
Previously the `detailLabel` was only used for `.appVersion` so we no longer need it here.